Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecoratorTest.java

```diff
@@ -42,106 +47,125 @@
-            }
-        };
-        
-        when(mockPropAccessor.getPropertyNames(NhincConstants.GATEWAY_PROPERTY_FILE)).thenReturn(new HashSet<String>());
-        
-        headerDecorator.configure();
-        
-        validateConfiguration(headerDecorator);
-    }
-    
-    @Test
-    public void testConfigure_WithPropertyValues() throws PropertyAccessException {
-        CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
-                new TestServicePortDescriptor());
-
-        ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
-        
-        final PropertyAccessor mockPropAccessor = mock(PropertyAccessor.class);
-
-        AssertionType assertion = new AssertionType();
-        
-        HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion) {
-            
-            @Override
-            protected PropertyAccessor getPropertyAccessor() {
-                return mockPropAccessor;
-            }
-        };
-        
-        final String customHeaderName = "customName";
-        final String customHeaderValue = "customValue";
-        Set<String> headerNames = new HashSet<>();
-        headerNames.add(NhincConstants.CUSTOM_HTTP_HEADERS + "." + customHeaderName);
-        
-        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.KEEP_ALIVE_PROP)).thenReturn("True");
-        when(mockPropAccessor.getPropertyNames(NhincConstants.GATEWAY_PROPERTY_FILE)).thenReturn(headerNames);
-        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.CUSTOM_HTTP_HEADERS + "." + customHeaderName)).thenReturn(customHeaderValue);
-        
-        headerDecorator.configure();
-        
-        validateConfiguration(headerDecorator);
-        BindingProvider bp = (BindingProvider) headerDecorator.getPort();
-        Map<String, List<String>> bpMap = (Map<String, List<String>>) bp.getRequestContext().get(NhincConstants.CUSTOM_HTTP_HEADERS);
-        assert(bpMap.containsKey("customName"));
-    }
-    
-    private void validateConfiguration(HttpHeaderServiceEndpointDecorator headerDecorator) {
-        assertEquals(headerDecorator.getHTTPClientPolicy().getConnection(), ConnectionType.KEEP_ALIVE);
-        
-        BindingProvider bp = (BindingProvider) headerDecorator.getPort();
-        Map<String, List<String>> bpMap = (Map<String, List<String>>) bp.getRequestContext().get(NhincConstants.CUSTOM_HTTP_HEADERS);
-        assertNotNull(bpMap);
-        assertEquals(bpMap.size(), 1);
-    }
+	public HttpHeaderServiceEndpointDecoratorTest() {
+	}
+
+	@Before
+	public void setUp() {
+
+	}
+
+	@After
+	public void tearDown() {
+	}
+
+	@Test
+	public void testConfigure_WithAssertionValues() throws PropertyAccessException {
+		CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(new TestServicePortDescriptor());
```

Comment:
Did u follow https://connectopensource.atlassian.net/wiki/display/CONNECTWIKI/Coding+templates+and+formatters for code format?  I see a lot of tab instead of space.